consensus/bor/bor.go

```diff
@@ -138,6 +138,13 @@ var (
 	errRecentlySigned = errors.New("recently signed")
 )
 
+type ActiveSealingOp struct {
+	ctx    context.Context
+	cancel context.CancelFunc
```

Comment:
Why it needs `ctx` and `cancel`? Why not just `blockNumber` and `shouldSeal`?

Reply:
Aah I don't need to use a cancellable object at all. Changed.